_posts/2019-08-09-getting-started.md

```diff
@@ -8,7 +8,7 @@ tags: [getting started]
 
 ## Prerequisites
 
-Follow the [Jekyll Docs](https://jekyllrb.com/docs/installation/) to complete the installtion of basic environment (`Ruby `, `RubyGems` and `Bundler`). 
+Follow the [Jekyll Docs](https://jekyllrb.com/docs/installation/) to complete the installation of basic environment (`Ruby `, `RubyGems` and `Bundler`). 
```

Comment:
I'm very grateful to you for the PR!

Would you please remove one more blank character within "\`Ruby \`"? I mean change it to "\`Ruby\`"

Reply:
I've changed it within your commit. Thanks again!